org.sonatype.m2e.egit/src/org/sonatype/m2e/egit/internal/EgitScmHandler.java

```diff
@@ -101,6 +101,30 @@ protected String normalizeUri(String uri) throws URISyntaxException {
       throw new URISyntaxException(uri, "Invalid git URI");
     }
 
+    // Normalize uri like
+    // git@github.com:errai/errai.git/errai-common
+    // into
+    // git://github.com/errai/errai.git
+    // These 3 distinctions may happen independently in other cases.
+    String gitPrefix = "git://";
```

Comment:
It looks like git@ is actually an ssh:// url which, in github's case, only requires you to have a private key that matches with a ssh public key signature in github settings.